contract_compassion/views/contract_view.xml

```diff
@@ -83,6 +83,7 @@
                         <field name="group_freq"/>
                         <field name="months_paid"/>
                         <field name="last_paid_invoice_date"/>
+                        <field name="last_letter" invisible="1"/>
```

Comment:
It's not needed to add it in the view if you keep it invisible. Either remove it or keep it visible.